service_storagefacility/app/repositories/dao/storage/StorageUnitDao.scala

```diff
@@ -101,9 +101,19 @@ class StorageUnitDao @Inject() (
   /**
    * TODO: Document me!!!
    */
-  def getById(mid: MuseumId, id: StorageNodeDatabaseId): Future[Option[StorageUnit]] = {
+  def getById(
```

Comment:
These methods have not had their signatures updated. Don't remember if I said that was OK, or if they were just forgotten in the process:

* getNodeById
* getStorageTypesInPath
* findRootNodes
* getChildren
* getStorageNodeByName